Tests fail because of node version incompatibility

Take a look at https://github.com/mdn/interactive-examples/actions/runs/4145722747/jobs/7170466565
```
Run yarn --frozen-lockfile
yarn install v1.22.19
[1/[5](https://github.com/mdn/interactive-examples/actions/runs/4145722747/jobs/7170466565#step:4:6)] Validating package.json...
error interactive-examples@1.0.3: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "1[6](https://github.com/mdn/interactive-examples/actions/runs/4145722747/jobs/7170466565#step:4:7).19.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

Related to #2425 

@queengooborg  @caugner

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -13,7 +13,7 @@ jobs:
       - name: Setup Node.js environment
         uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: 18
           cache: 'yarn'
 
       - name: Install all yarn packages

